src/iop/useless.c

```diff
@@ -41,17 +60,24 @@ typedef struct dt_iop_useless_params_t
   // stored in self->params and self->default_params
   // also, since this is stored in db, you should keep changes to this struct
   // to a minimum. if you have to change this struct, it will break
-  // users data bases, and you should increment the version
-  // of DT_MODULE(VERSION) above!
-  int checker_scale;
-  float factor;
+  // users data bases, and you have to increment the version
+  // of DT_MODULE(VERSION) above and provide a legacy_params upgrade path!
+  // Tags in the comments get picked up by the introspection framework and are
```

Comment:
The above 9 lines need a bit more sentence structure. I'm not sure where a sentence ends and the next begins. Here's what I think it says (with a couple of corrections thrown in): 

"The variables in this struct are stored in the database. Make sure none of these variables depend on temporary memory (pointers etc.). This struct is stored in `self->params` and `self->default_params`. Please keep changes to this struct to a minimum. In order to provide backward-compatibility with old edits, each time the struct changes you must increment the version of DT_MODULE(VERSION) above and provide a method to upgrade old parameters in legacy_params."

Reply:
I think I forgot to commit a comment here. I think I suggested:
"This struct defines the layout of self->params and self->default_params."

Reply:
That makes more sense